dashboard/app/views/levels/_content.html.haml

```diff
@@ -4,15 +4,15 @@
 - app, data, content_class, source_level, hide_header, postcontent = %i(app data content_class source_level hide_header postcontent).map{ |x| local_assigns[x] }
 
 - unless data['title'].blank? || hide_header
-  %h1!= string_or_image(app, data['title'], source_level) unless app == 'external'
+  %h1!= source_level.localized_text(data['title']) unless app == 'external'
 %div.content-level{class: (content_class unless content_class.blank?)}
   - unless app == 'external'
     - unless data['content1'].blank?
-      %p.content.content1!= string_or_image(app, data['content1'], source_level)
+      %p.content.content1!= render_multi_or_match_content(data['content1'], source_level)
```

Comment:
I like the increased specificity with the rename from `string_or_image` to `render_multi_or_match_answer` and I think the name is accurate, however it's a bit tricky to see this since we include this partial in the following places before excluding this block due to app type `external`:
* https://github.com/code-dot-org/code-dot-org/blob/e4b95e0595d0cf7e08faf1a4d7f2e53599ad68b1/dashboard/app/views/levels/instructions.haml#L48
* https://github.com/code-dot-org/code-dot-org/blob/27a0d28d811df56d3c0900a155cfe7d4d5ad7476/dashboard/app/views/levels/_external.html.haml#L19

You don't have to be on the hook for doing this, but would a reasonable continuation of this refactor be to extract new `_external_content` and `_multi_match_content` partials from this `_content` partial, so that this partial doesn't have to "reject" the parts of it that are specific to multi and match?

Reply:
Oh, good point! I'll add an x-team item to track this.

Reply:
[added here](https://codedotorg.atlassian.net/browse/XTEAM-154)